### PR TITLE
Search Input Clears After You Click Stop Searching

### PR DESCRIPTION
**Pull request by trautlein**
I also fixed some spelling errors and added an error in another file

### CODE REVIEW

File: app/components/NavView.jsx [hunk 1]
```diff
@@ -59,21 +62,7 @@ const styles = {
     fontFamily: 'droid sans',
     textDecoration: 'none',
     color: 'white'
-  },
```
Comment (trautlein) [inline app/components/NavView.jsx:62]: Moved to searchView

File: app/components/NavView.jsx [hunk 2]
```diff
@@ -12,9 +12,12 @@ class NavView extends React.Component {
         <span style={styles.greenCast}>greenCast</span>
         <a style={styles.logout} href='/logout'>Logout <i className="fa fa-sign-out" ariaHidden="true"></i></a>
         <div style={styles.search}>
-          <Search handleSearchInputChange={this.props.handleSearchInputChange}/>
+          <Search
```
Comment (trautlein) [inline app/components/NavView.jsx:15]: Passed in these values when I moved stop searching button inside searchview

File: app/components/SearchView.jsx [hunk 3]
```diff
@@ -16,6 +16,11 @@ class SearchView extends React.Component {
     });
   }
 
+  innerStopSearching() {
+    this.props.stopSearching();
```
Comment (trautlein) [inline app/components/SearchView.jsx:20]: I wasted too much time here!

**Review by chasestarr — APPROVED** [review]
Awesome changes!

**Pull request merged**